uwblueprint/supportive-housing
# Kelly/get buildings route

## Notion task link

[Get Buildings Route](https://www.notion.so/uwblueprintexecs/Get-Buildings-Route-e5b2d8b0d0004b2e8754d133fa4ed3a5?pvs=4)



## Implementation description
* added GET endpoint to get all buildings from DB
* replaced hardcoded `BUILDINGS` variable with the buildings fetched from the database



## Steps to test
1. Check if you're able to see the building options on log filter, add log, edit log, add resident, edit resident
2. Check if the functionality works for the places mentioned above

## Checklist
